cert-manager/cert-manager-webhook-role-binding/base/role-binding.yaml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
nit - not needed

Reply:
I'll remove them throughout this package, wherever I find them in the beginning of the file.